test/REXUniswapV3Market.test.ts

```diff
@@ -1027,26 +782,28 @@ describe('REXUniswapV3Market', () => {
           receiver: market.address,
           superToken: ricochetUSDCx.address,
           flowRate: inflowRateUsdc,
-          userData: ethers.utils.defaultAbiCoder.encode(['string'], ['carl']),
           shouldUseCallAgreement: true,
           overrides,
         })
         .exec(aliceSigner)
 
+      console.log('Alice USDCx stream created')
+
       // Advance time to allow for some tokens to accumulate in the market
       await increaseTime(TEST_TRAVEL_TIME)
 
-      // Bob opens a USDC stream to REXMarket, triggers a distribute
-      await sf.cfaV1
-        .createFlow({
-          sender: bobSigner.address,
-          receiver: market.address,
-          superToken: ricochetUSDCx.address,
-          flowRate: inflowRateUsdc,
-          shouldUseCallAgreement: true,
-          overrides,
-        })
-        .exec(bobSigner)
+      // // Bob opens a USDC stream to REXMarket, triggers a distribute
+      // await sf.cfaV1
+      //   .createFlow({
+      //     sender: bobSigner.address,
+      //     receiver: market.address,
+      //     superToken: ricochetUSDCx.address,
+      //     flowRate: '100',
+      //     shouldUseCallAgreement: true,
+      //     overrides,
+      //   })
+      //   .exec(bobSigner)
+      // console.log('Bob USDCx stream created');
```

Comment:
Should we remove it instead of commenting?

Reply:
Yes, I can do that